content/docs/zero/install.mdx

```diff
@@ -34,7 +34,7 @@ Pomerium requires an internet-accessible server and the ability to bind port 443
 First, install Pomerium into your Kubernetes cluster:
 
 ```bash
-kubectl apply -k github.com/pomerium/pomerium/k8s/zero\?ref=v0.26.0
+kubectl apply -k github.com/pomerium/pomerium/k8s/zero\?ref=v0.27.0
```

Comment:
I think this one may need a one-time change to be `ref=main`?

@wasaga please let us know if that's correct.

Reply:
(Oh, didn't see you responded already, please ignore.)